integration/cli-tests/src/test/java/org/wso2/micro/integrator/cli/LoginLogoutTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.micro.integrator.cli;
+
+import java.io.IOException;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import util.TestUtils;
+
+import static util.TestUtils.getMIBuildPath;
+
+public class LoginLogoutTest extends AbstractCliTest {
+
+    @Test
+    public void loginTest() throws IOException {
+        List<String> loginOutput = super.login();
+        Assert.assertTrue(loginOutput.get(0).contains("Login successful for remote"));
+    }
+
+    @Test(dependsOnMethods = {"loginTest"})
```

Comment:
Better if we don't have to depend on methods, rather make methods self contained.

Reply:
Updated.